chapter_CI_hypothesis/CI_and_hypothesis_test.Rmd

```diff
@@ -0,0 +1,1088 @@
+
+--
+
+## Today: Deeper dive into ***statistical inference*** <sup>1</sup>
+
+.footnote[
+[1]: This lecture is based on the wonderful [confidence interval](https://moderndive.com/8-confidence-intervals.html) and [hypothesis testing](https://moderndive.com/9-hypothesis-testing.html) chapters of [ModernDive](https://moderndive.com/)]
+
+* Process of making claims about a population based on information from a sample
+
+* *Confidence intervals*: providing plausible ***range*** of values
+
+* *Hypothesis testing*: comparing statistics between groups
+
+---
+
+background-image: url(https://media.giphy.com/media/hXG93399r19vi/giphy.gif)
+background-position: 18% 45%
+background-size: 450px
+
+# Back to reality (there goes gravity `r emo::ji("wink")`)
+
+.pull-right[
+
+* In real life we only get to take ***one*** sample from the population (not ***1000***!).
+
+* Also, we obviously don't know the true population parameter, that's what we are interested in!
+
+* So what on earth was all of this good for? Fun only?! `r emo::ji("anguished")`
+
+]
+
+--
+
+<br>
+<br>
+
+* Even unobserved, we ***know*** that the sampling distribution does exist, and even better, we know how it behaves!
+
+* Let's see what we can do with this...
+
+---
+
+layout: false
+class: title-slide-section-red, middle
+
+# Confidence Intervals
+
+---
+
+layout: true
+
+<div class="my-footer"><img src="../img/logo/ScPo-shield.png" style="height: 60px;"/></div> 
+
+---
+
+# From Point Estimates to Confidence Intervals
+
+* Until now, we have only estimated ***point estimates*** from our samples: *sample means*, *sample proportions*, *regression coefficients*, etc.
+
+--
+
+* We know that this ***sample statistic*** differs from the ***true population parameter*** due to ***sampling variation***. 
+
+--
+
+* Rather than a point estimate, we could give a ***range of plausible values*** for the population parameter.
+
+--
+
+* This is precisely what a ***confidence interval*** (CI) provides.
+
+---
+
+# Constructing Confidence Intervals 
+
+There are several approaches to constructing confidence intervals:
+    
+  1. *Theory*: use mathematical formulas (***Central Limit Theorem***) to derive the sampling distribution of our point estimate under certain conditions.
+
+--
+
+  1. *Simulation*: use the ***bootstrapping*** method to *reconstruct* the sampling distribution of our point estimate.
+    
+--
+    
+We'll focus on simulation to give you the intuition and come back to the maths approach next week.
+
+---
+
+# Back to Pasta
+
+* As in real life, imagine we had access to *only one random sample* from our bowl of past.
+
+--
+
+* How could we study the effect of sampling variation with a single sample? `r emo::ji("point_right")` ***bootstrap resampling with replacement***!
+
+--
+
+* Let's start by drawing one random sample of size $n = 50$ from our bowl.
+
+```{r, echo = FALSE}
+bowl <- read.csv("https://www.dropbox.com/s/qpjsk0rfgc0gx80/pasta.csv?dl=1")
```

Comment:
you have this in the package included:
```R
> data("pasta_jar",package = "ScPoEconometrics")
> ls()
[1] "pasta_jar"
```

Reply:
this is actually a different pasta bowl dataset: the census of the pasta I had, I didn't know that the package came with your pasta_jar data.